process/info.go

```diff
@@ -107,13 +108,27 @@ func getProjectDetails(path string) (string, []tide.InfoDetails, error) {
 		return "", nil, err
 	}
 
+	// If the Message says its a theme, lets trust that it is a theme.
+	var isTheme bool
+	if msg.ProjectType == "theme" {
+		var pt string
+		pt, details, err = extractHeader(path + "/style.css")
+		isTheme = pt == "theme"
+		found = true
+	}
+
 	// Traverse files and scan for headers.
-	for _, f := range files {
-		projectType, details, err = extractHeader(path + "/" + f.Name())
-		if err == nil {
-			found = true
-			break
+	if ! isTheme {
```

Comment:
This doesn't solve the issue where multiple headers are found in a plugin. How do we know we're not getting the same false info for a plugin that has the TGM or other single file plugin in the project root?

Reply:
I'm reverting this, I have a better way that will allow for multiple headers.  Also, filename was a bad idea.